data/helpers.d/psql

```diff
@@ -289,6 +290,6 @@ ynh_psql_test_if_first_run() {
 		yunohost service add postgresql --log "$logfile"
 
 		systemctl enable postgresql
-		systemctl reload postgresql
+		ynh_systemd_action --service_name=postgresql --action=reload
```

Comment:
Kinda unrelated but just to say that this stuff about the manipulating postgresql here is a bit weird / misleading : the "real" postgresql service seems to be `postgresql@9.6-main` (or something like this) compared to just `postgresql` service which is a different thing ... Dunno if that's an issue or not but would be nice if people are aware of this :P